packages/server/src/oauth/keys.test.ts

```diff
@@ -25,25 +22,6 @@ describe('Keys', () => {
     await shutdownApp();
   });
 
-  test('Init keys', () =>
```

Comment:
This is a very old test, which predates almost everything.  All code paths here are already covered by `initAppServices()` which runs in `seed.test.ts`, and most other server tests.